src/components/SubmitButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+interface ButtonProps {
+  loading: boolean;
+  message: string;
+}
+const SubmitButton = ({ loading, message }: ButtonProps) => {
+  return (
+    <button
+      type="submit"
+      className="px-40 py-5 bg-transparent hover:bg-blue-500 text-blue-700 font-semibold hover:text-white border border-blue-500 hover:border-transparent rounded"
+      disabled={loading}
+    >
+      {loading ? (
+        <>
+          <svg
```

Comment:
Add this SVG as a file in your public folder under an `icons` folder to be served statically, then import into this file as a component. Search for how you can import SVG files into React components.